### DISCUSSION TITLE
Partial updates

### DISCUSSION THREAD

**Discussion by nerdoc** [tetra-framework/tetra#58] [Ideas]
Sometimes, especially on big components, not the whole component HTML must be rerendered, but just a small part. It could be helpful if only this part is rendered and morphed into the DOM at the client on demand.

This could be done using a manual parameter on `update(partial="#somediv_id")`, or better directly using a separated `update_partial(id="somediv_id")` in a public method.

**Comment by gsxdsm** [comment]
Love this. Something I can look into
